project/Common.scala

```diff
@@ -15,7 +15,7 @@ object Common extends AutoPlugin {
 
   val AkkaVersion = sys.env.get("AKKA_SERIES") match {
     case Some("2.5") => "2.5.13"
-    case _ => "2.4.18"
+    case _ => "2.5.13"
   }
```

Comment:
This whole `match` expression could be replaced by `"2.5.13"`, but I wanted to preserve the logic in case we support multiple `AKKA_SERIES` in the future (say 2.5 and 2.6).

Reply:
Although if my reasoning that it is OK to require some specific minimal Akka version for each akka-stream-contrib release is correct, then we can get rid of this `AKKA_SERIES` environment variable for good.

Reply:
Because of the binary compatibility guarantees, it is possible to depend on the latest version from the series and use the latest features and bugfixes. Binary compatibilty guarantees that older code will work fine with a later Akka version without recompiling.

I say lets get rid of the env variable all together for now, and use the latest Akka version.

Reply:
That statement is wrong or at least misleading. Binary backwards compatibility means that an application using this library can use the latest Akka version 2.5.21 even though this library was built with Akka 2.5.13. The opposite is not true. If this library was built with Akka 2.5.21 an application is not guaranteed to be able to run with an older version of Akka, like 2.5.20. In other words, the version defined here specifies the minimum required version.

For akka-stream-contrib I don't have any opinions about what version that is supported. It's perfectly fine to use latest Akka (and definitely no need to support 2.4.x).

Reply:
Right. So any time akka-stream-contrib bumps up its minimal required Akka version, users will have to bump up their Akka version as well. That, however, should not cause them any trouble.

Reply:
Yea, sorry for being a bit vague here. Thanks @patriknw for clearing it up.